mawabot/client.py

```diff
@@ -78,7 +79,7 @@ def run(self):
         self.add_cog(Reloader(self))
         logger.info('Loaded cog: Reloader')
 
-        files = [PY_FILE_REGEX.sub('', file) for file in listdir('mawabot/cogs') if '.py' in file]
+        files = [cog for cog in listdir('mawabot/cogs') if isdir(f'mawabot/cogs/{cog}')]
```

Comment:
Good news: This fixes the issue with Python trying to import vim swap files.
Bad news:

```py
Traceback (most recent call last):
  File "/home/ammon/Git/mawabot/mawabot/client.py", line 88, in on_ready
    self.load_extension(f'mawabot.cogs.{file}')
  File "/home/ammon/.local/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 684, in load_extension
    raise discord.ClientException('extension does not have a setup function')
discord.errors.ClientException: extension does not have a setup function
```

We should add a check that can prevent this, by checking if the name starts with an underscore.  
I've pushed my recommended changes. (d9df1e1 and 0294d1d)